src/app/components/tool-bar/tool-bar.component.scss

```diff
@@ -0,0 +1,19 @@
+.navbar {
+  background-color: rgb(136,89,83);
+  display: flex;
+  justify-content: space-between;
+}
+
+.navbar__header{
+  color: white;
+}
+
+.navbar__logo {
+  width: 55px;
+  margin: 5px;
+}
+
+.navbar__links {
+  margin-left: 10px;
+  border: 1px solid black;
+}
```

Comment:
```suggestion
.navbar {
  background-color: rgb(136,89,83);
  display: flex;
  justify-content: space-between;
  
    &__header{
       color: white;
    }
    
    &__logo {
      width: 55px;
      margin: 5px;
   }
   
   &__links {
    margin-left: 10px;
    border: 1px solid black;
  }

 }
```

Reply:
css files are not in proper format, please refer to the above changes and update the rest of the files in the same format.

Reply:
Changed the format of all the scss files to BEM convention.